File: python/8 kyu/372.py
# https://www.codewars.com/kata/577a6e90d48e51c55e000217/train/python

# Passed

def hotpo(n):
    counter = 0
    while n != 1:
        counter += 1
        if n % 2 == 0:
            n = n / 2
        else:
            n = 3 * n + 1
            
    return counter

output = hotpo(23)
print(output)

File: nodejs/7 kyu/372.js
// https://www.codewars.com/kata/5b39e91ee7a2c103300018b3/train/javascript

// Passed

function removeConsecutiveDuplicates(string) {
  let wordsToJoin = [];
  let lastWord;
  for (const word of string.split(" ")) {
    if (word === lastWord) {
      continue;
    }

    lastWord = word;
    wordsToJoin.push(lastWord);
  }

  let result = wordsToJoin.join(" ");
  return result;
}

const output = removeConsecutiveDuplicates(
  "alpha beta beta gamma gamma gamma delta alpha beta beta gamma gamma gamma delta",
);
console.log(output);